geode-core/src/main/java/org/apache/geode/management/internal/i18n/CliStrings.java

```diff
@@ -523,11 +523,16 @@ public class CliStrings {
   public static final String BACKUP_DISK_STORE_MSG_BACKED_UP_DISK_STORES =
       "The following disk stores were backed up successfully";
   public static final String BACKUP_DISK_STORE_MSG_OFFLINE_DISK_STORES =
-      "The backup may be incomplete. The following disk stores are not online";
+      "The backup may be incomplete. The following disk stores are failed";
   public static final String BACKUP_DISK_STORE_MSG_HOST = "Host";
   public static final String BACKUP_DISK_STORE_MSG_DIRECTORY = "Directory";
   public static final String BACKUP_DISK_STORE_MSG_UUID = "UUID";
   public static final String BACKUP_DISK_STORE_MSG_MEMBER = "Member";
+  public static final String BACKUP_DISK_STORE_MSG_FAILED_REASON = "Failed Reason";
+  public static final String BACKUP_DISK_STORE_MSG_FAILED_REASON_1 = "Disk store is offline";
+  public static final String BACKUP_DISK_STORE_MSG_FAILED_REASON_2 = "Insufficient permission";
+  public static final String BACKUP_DISK_STORE_MSG_FAILED_REASON_3 = "No space left";
+  public static final String BACKUP_DISK_STORE_MSG_FAILED_REASON_4 = "Other disk reasons";
```

Comment:
I would rename these constants to significant names instead of _#. Example: `BACKUP_DISK_STORE_MSG_FAILED_REASON_OFFLINE`

Reply:
it's ok for me